src/main/java/com/skynest/uitesting/tests/BaseTest.java

```diff
@@ -0,0 +1,100 @@
+package com.skynest.uitesting.tests;
+
+import com.github.javafaker.Faker;
+import com.skynest.uitesting.pages.LoginPage;
+import com.skynest.uitesting.pages.RegistrationPage;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.firefox.FirefoxDriver;
+import org.openqa.selenium.ie.InternetExplorerDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+
+import java.time.Duration;
+
+import static com.skynest.uitesting.utils.BaseHelper.randomEmail;
+
+public class BaseTest {
+    public static final int TIMEOUT = 20;
+    public WebDriver driver;
+    public WebDriverWait wait;
+    public String browser = "chrome";
+    public Faker faker = new Faker();
+    public String firstName;
+    public String lastName;
+    public String emailAddress;
+    public String phoneNumber;
+    public String homeAddress;
+    public String password;
+    public String confirmPassword;
+    protected LoginPage loginPage;
+    protected RegistrationPage registrationPage;
+
+    @BeforeClass
+    public void setUserCredentials() {
+        firstName = faker.name().firstName();
+        lastName = faker.name().firstName();
+        emailAddress = randomEmail();
+        phoneNumber = "23458752345";
+        homeAddress = faker.address().fullAddress();
+        password = "Selenium22";
+        confirmPassword = password;
+    }
+
+    @BeforeMethod
+    public void beforeMethod() {
+        switch (browser.toLowerCase()) {
+            case "chrome":
+                System.setProperty("webdriver.chrome.driver", "src/test/driver/chromedriver.exe");
```

Comment:
On this line, since you're using webdriver manager, you should be able to just replace this line with: `WebDriverManager.chromedriver().setup();` and add the approprate statements to all other switch cases, and it should work